homeassistant/components/zwave/__init__.py

```diff
@@ -600,14 +601,36 @@ def start_zwave(_service_or_event):
     return True
 
 
-class ZWaveDeviceEntity:
+class ZWaveDeviceEntity(Entity):
```

Comment:
Is there a reason that `ZWaveDeviceEntity` did not inherit `Entity` already? From what I found every child instance pulled in `Entity` anyway.